controller/controller.go

```diff
@@ -207,7 +207,11 @@ func UpdateExtensions(w http.ResponseWriter, r *http.Request) {
 	if len(updateRequest) == 1 {
 		_, ok := AllExtensionsMap[updateRequest[0].ID]
 		if !ok {
-			http.Redirect(w, r, "https://update.googleapis.com/service/update2?braveRedirect=true", http.StatusTemporaryRedirect)
+			queryString := "braveRedirect=true"
+			if len(r.URL.RawQuery) != 0 {
```

Comment:
there is this other place where there could be a problem if the query is empty: https://github.com/brave/go-update/pull/9/files#diff-18de8fd9ec37aa97da8831f9408ce6aeR153. it hasn't caused apparent issues yet so might be ok.

Reply:
@diracdeltas I did see that other reference, but it's used only when there is an x parameter which means a query string will always exist there.